src/components/MovieDetails/MovieDetails.js

```diff
@@ -13,8 +13,9 @@ class MovieDetails extends Component {
   }
 
   componentDidMount = () => {
-    apiFetchRatings(this.props.userInfo.id)
-      .then(data => this.props.fetchUserRatings(data.ratings))
+    this.props.userInfo.id
+      && apiFetchRatings(this.props.userInfo.id)
+          .then(data => this.props.fetchUserRatings(data.ratings))
```

Comment:
Had to add this conditional to only fetch the ratings if the user is logged in. Otherwise we get errors.

Reply:
Nice! This will help reduce problems of errors. Good catch!

Reply:
So does this operate like an if block just by saying this.props.userInfo.id && apiFetchRatings? I just want to make sure I understand the syntax going on here.

Reply:
From my understanding, if there is and user ID, then run the fetch call.  That is what the method is doing.  You can correct me if I am wrong HArry.

Reply:
Thanks! I think I understand what it does. I still don't really understand how this syntax does that without an if or try or something.

Reply:
Yep, I think you both are right. It's kind of like shorthand for an if statement with only one outcome. If userInfo.id exists then run the code on the other side. If not then it will not run the code on the other side of &&.

Reply:
Cool. Thanks for explaining!